File: programmers/Lv_2/JeongJunKwon/숫자의 표현.java
class Solution {
  public int solution(int n) {
      int answer = 0;
      int total = 0;
      for(int i = 1; i <= n; i++) {
          int index = i;
          while(true) {
              total += index++;
              if(total == n) {
                  answer++;
                  total = 0;
                  break;
              } else if (total > n) {
                  total = 0;
                  break;
              }
          }
      }
      return answer;
  }
}

File: programmers/Lv_2/HwangJunghoon/숫자의 표현.py
def solution(n):
    # answer : 자기 자신을 포함시켜서 1부터 시작
    answer,count=1,1
    while(count<n):
        half=0
        for i in range(count,n+1):
            half+=i
            if half==n:
                answer+=1
                count+=1
                break
            elif half>n:
                count+=1
                break
    return answer